dom.bs

```diff
@@ -4533,33 +4533,82 @@ each <a for=tree>descendant</a> <a>exclusive <code>Text</code> node</a> <var>nod
 <a lt="Other applicable specifications">Specifications</a> may define
 <dfn export id=concept-node-clone-ext>cloning steps</dfn> for all or some <a for=/>nodes</a>. The
-algorithm is passed <var>copy</var>, <var>node</var>, <var>document</var>, and an optional
-<i>clone children flag</i>, as indicated in the <a lt="clone a node">clone</a> algorithm.
+algorithm is passed <var ignore>node</var>, <var ignore>copy</var>, and <var ignore>subtree</var> as
+indicated in the <a>clone a node</a> algorithm.
 
-<p class=note>HTML defines <a>cloning steps</a> for <{script}> and <{input}>
-elements. SVG ought to do the same for its <{script}> elements, but does not call this out
-at the moment.
+<p class=note>HTML defines <a>cloning steps</a> for several elements, such as <{input}>, <{script}>,
+and <{template}>. SVG ought to do the same for its <{script}> elements, but does not.
+</div>
 
-<p>To <dfn export id=concept-node-clone lt="clone a node" local-lt="clone">clone</dfn> a
-<var>node</var>, with an optional <var>document</var> and <i>clone children flag</i>, run these
-steps:
-<!-- This algorithm is used by dom-Node-cloneNode, dom-Document-importNode,
-dom-Range-extractContents, dom-Range-cloneContents -->
+<div algorithm="clone a node">
+<p>To <dfn export id=concept-node-clone>clone a node</dfn> given a <a for=/>node</a> <var>node</var>
+and an optional <a for=/>document</a> <dfn export for="clone a node"><var>document</var></dfn>
+(default <var>node</var>'s <a for=Node>node document</a>), boolean
+<dfn export for="clone a node"><var>subtree</var></dfn> (default false), and
+<a for=/>node</a>-or-null <dfn export for="clone a node"><var>parent</var></dfn> (default null):
 
 <ol>
- <li><p>If <var>document</var> is not given, then set <var>document</var> to <var>node</var>'s
- <a for=Node>node document</a>.
+ <li><p><a for=/>Assert</a>: <var>node</var> is not a <a for=/>document</a> or <var>node</var> is
+ <var>document</var>.
 
- <li><a for=/>Assert</a>: <var>node</var> is not a <a for=/>document</a> or <var>node</var> is
+ <li><p>Let <var>copy</var> be the result of <a>cloning a single node</a> given <var>node</var> and
  <var>document</var>.
 
+ <li><p>If <var>parent</var> is non-null, then <a for=/>append</a> <var>copy</var> to
+ <var>parent</var>.
+
+ <li><p>Run any <a>cloning steps</a> defined for <var>node</var> in
+ <a>other applicable specifications</a> and pass <var>node</var>, <var>copy</var>, and
+ <var>subtree</var> as parameters.
+
+ <li><p>If <var>subtree</var> is true, then for each <var>child</var> of <var>node</var>'s
+ <a for=tree>children</a>, in <a>tree order</a>: <a>clone a node</a> given <var>child</var> with
+ <a for="clone a node"><i>document</i></a> set to <var>document</var>,
+ <a for="clone a node"><i>subtree</i></a> set to <var>subtree</var>, and
+ <a for="clone a node"><i>parent</i></a> set to <var>copy</var>.
+
+ <li>
+  <p>If <var>node</var> is an <a for=/>element</a>, <a for=Element>shadow host</a>, and its
```

Comment:
This insertion of "an element" makes the clause not really seem to parse properly (as English) anymore.  I'm not quite sure what the change relative to the old wording was intended to be, but I can't tell what the new wording means.